docs/usage/openapi-definition-resolver.md

```diff
@@ -51,6 +51,36 @@ SwaggerClient.resolve({ url: 'https://raw.githubusercontent.com/swagger-api/swag
 > *__Note:__ you can provide `Swagger.resolve` both `spec` and `url` options. In that case
 `spec` will always be preferred and `url` option will be ignored.* 
 
+#### Authentication
+
+When JSON-References points to a remote document protected by authentication,
+`requestInterceptor` option can be used to provide e.g. an authentication header.
```

Comment:
`provide`... what? recommend specifying additional examples

Reply:
Changed it to

> When JSON-References points to a remote document protected by authentication,
`requestInterceptor` option can be used to provide request authentication (e.g. by setting `Authentication` header).

Let me know if it needs some additional tweaking.